src/multivariate.jl

```diff
@@ -151,65 +171,92 @@ function _logpdf(d::TuringMvLogNormal, x::AbstractVector{T}) where {T<:Real}
     return insupport(d, x) ? (_logpdf(d.normal, log.(x)) - sum(log.(x))) : -T(Inf)
 end
 function _logpdf(d::TuringMvLogNormal, x::AbstractMatrix{T}) where {T<:Real}
-    if all(insupport(d, x))
+    insupp = map(1:size(x, 2)) do i 
+        insupport(d, x[:, i])
+    end
```

Comment:
Is it actually necessary to allocate an array? I guess one could just check `if all(i -> insupport(d, view(x, :, i)), 1:size(x, 2))` or something similar.

Reply:
We need `insupp` later in case not all are in the support.